source/rfc/versioning.rst

```diff
@@ -0,0 +1,54 @@
+Upstream Components Versioning 
+============================== 
+
+Purpose of this document       
+------------------------       
+  
+This document contains design for implementation of external/upstream components usage and mirroring in Contrail 5.0+ project.
+  
+Problem
+-------
+
+Upstream versions of operating systems and other software (e.g. OpenStack) change in schedules different than that of Contrail's. Red Hat family of operating systems replaces a previous version with new one and drops the support for older ones. There are multiple combinations of supported OpenStack and CentOS/Red Hat releases and they all have to be tracked.
+Using official repositories makes CI setup prone to network failures.
+    
+Solution
+--------
+
+Life cycle for external/upstream components has to be tracked in order to accomodate for changes in operating systems' and other software's versioning.
```

Comment:
`Lifecycle of`
IMO, not sure about the space tho

Reply:
Looks good to me